oak-lucene/src/main/java/org/apache/jackrabbit/oak/plugins/index/lucene/LucenePropertyIndex.java

```diff
@@ -1589,6 +1604,22 @@ private static Query newDepthQuery(String path, PlanResult planResult) {
 
         @Override
         public List<Facet> getFacets(int numberOfFacets, String columnName) throws IOException {
+            if (!CACHE_FACET_RESULTS) {
+                LOG.trace(CACHE_FACET_RESULTS_NAME + " = " + CACHE_FACET_RESULTS + " getting uncached results for columnName = " + columnName);
```

Comment:
Good idea to use LOG.trace! In theory, it would be slightly faster to use

    LOG.trace("{} = {}  getting uncached results for columnName = {}",  
            CACHE_FACET_RESULTS_NAME, CACHE_FACET_RESULTS, columnName);

Actually it would be sufficient to use

    LOG.trace("Uncached results for columnName = {}", columnName);